tests/bettertransformer/test_bettertransformer_vision.py

```diff
@@ -66,3 +67,20 @@ def prepare_inputs_for_class(self, model_id=None):
         processor = AutoProcessor.from_pretrained(model_id)
         inputs = processor(image, text, return_tensors="pt")
         return inputs
+
+
+class BetterTransformersFlavaTest(BetterTransformersTestMixin, unittest.TestCase):
+    r"""
+    Testing suite for Vision and Text Models - tests all the tests defined in `BetterTransformersTestMixin`
+    """
+    all_models_to_test = ALL_VISION_TEXT_MODELS_TO_TEST
```

Comment:
```suggestion
    all_models_to_test = ["ybelkada/tiny-random-flava"]
```
This is better I guess ;)